WordPress/src/main/java/org/wordpress/android/ui/domains/DomainRegistrationDetailsFragment.kt

```diff
@@ -264,18 +262,16 @@ class DomainRegistrationDetailsFragment : Fragment() {
         email_input.setText(domainContactInformation.email)
 
         if (!TextUtils.isEmpty(domainContactInformation.phone)) {
-            val phoneParts = domainContactInformation.phone!!.split(PHONE_NUMBER_CONNECTING_CHARACTER)
-            if (phoneParts.size == 2) {
-                var countryCode = phoneParts[0]
-                if (countryCode.startsWith(PHONE_NUMBER_PREFIX)) {
-                    countryCode = countryCode.drop(1)
-                }
-
-                val phoneNumber = phoneParts[1]
-
-                country_code_input.setText(countryCode)
-                phone_number_input.setText(phoneNumber)
-            }
+            country_code_input.setText(
+                    DomainPhoneNumberUtils.getPhoneNumberPrefixFromFullPhoneNumber(
```

Comment:
I'd do this mapping in the view model. It would mean creating a UI model of the DomainContactModel with the fields you need in the UI. I'd also make it an injected non-static class. Both of these will help you with testing :-).